### DISCUSSION TITLE
How to use with 2 different lists

### DISCUSSION THREAD

**Discussion by CheapSean123** [LanKuDot/game_modules#21] [General]
How would you use this script for 2 different lists? Especially if you want to display the content of centered object.
Left List is centered on 2, Right list is centered on 3
How would you display 2 3 on a text gameobject in unity?

**Comment by LanKuDot** [comment]
Do yo want to display two list content in the same list or two separated lists?
For the latter one, creating two lists and two list banks would work.

**Reply by CheapSean123** [reply]
I want to create two separated lists, how would you create two list banks?

**Reply by LanKuDot** [reply]
Create another new script such as `AnotherListBank.cs` and inherit from `BaseListBank`.
Then attach it to a gameobject and assign the reference to the list you want to show the content.
https://github.com/LanKuDot/game_modules/tree/master/CircularScrollingList#create-listbank

**Reply by CheapSean123** [reply]
Thanks, that seems to work, the only problem is that the first CircularList created seems to be controlling the other list index as well, so if i use the function and button to move the boxes, the text that is supposed to change indipendently between the two lists only changes when i touch the buttons attached to the first List. I attached a video to demonstrate this.

https://user-images.githubusercontent.com/84329716/118787377-119ae800-b893-11eb-8898-f5a085f379ac.mp4

**Reply by LanKuDot** [reply]
It seems that you invoked `GetCenteredContentID()` of the same list.
It should be something like:
```
var content1 = listBank1.GetListContent(list1.GetCenteredContentID());
var content2 = listBank2.GetListContent(list2.GetCenteredContentID());
```
